src/mon/Monitor.cc

```diff
@@ -3939,31 +3939,33 @@ void Monitor::handle_command(MonOpRequestRef op)
       overall[p.first] += p.second;
     }
     f->close_section();
-
+    if(!mgr.empty()){
     mgrmon()->count_metadata("ceph_version", &mgr);
     f->open_object_section("mgr");
     for (auto& p : mgr) {
       f->dump_int(p.first.c_str(), p.second);
       overall[p.first] += p.second;
     }
     f->close_section();
-
+    }
+    if(!osd.empty()){
```

Comment:
Similar changes required as mention for mgr block.

Reply:
There should be space between if keyword and starting parentheses here. Also there should be space between ending parentheses and curly bracket.